sdk/nodejs/iot/zMixins.ts

```diff
@@ -76,6 +76,16 @@ export class IoTHubEventSubscription extends appservice.EventSubscription<EventH
             connection: bindingConnectionKey,
         }];
 
+        pulumi.all([iotHub.fallbackRoute, iotHub.routes]).apply(([fallbackRoute, routes]) => {
+            if(fallbackRoute.enabled){
+                return;
+            }
+            if(routes && routes.length > 0){
```

Comment:
Nit: add spaces between `if (routes` and `0) {`

Reply:
fixed in 137e01167ee86367e57aa1ed13d65e9e281c7973